mobile/src/pages/feedPage.tsx

```diff
@@ -1,14 +1,22 @@
-import { View, ScrollView } from "react-native";
+import { View, ScrollView } from 'react-native';
 import BottomNavBar from "../components/BottomNavBar";
 import Card from "../components/Card";
 import TopNavBar from "../components/TopNavBar";
 
 export default function FeedPage() {
+
+  //# region Styles
+
+  const baseStyle = "items-center justify-center";
+  const viewStyle = "position-relative w-screen h-screen";
```

Comment:
Not your mistake, but `position-relative` doesn't actually do anything. `relative` is the class for that, but I think we can safely remove it.

Reply:
Also, if you can rename this file to `FeedPage.tsx` that would be great.